guides/schema/class_based_api.md

```diff
@@ -97,6 +96,36 @@ __A monkeypatch__, activated in global scope:
 GraphQL::DeprecatedDSL.activate
 ```
 
+### Connection fields & types
+
+There is no `connection(...)` method. Instead, connection fields are inferred from the type name.
+
+If the type name ends in `Connection`, the field is treated as a connection field.
+
+This default may be overridden by passing a `connection: true` or `connection: false` keyword.
+
+For example:
+
+```ruby
+# This will be treated as a connection, since the type name ends in "Connection"
+field :projects, Types::ProjectType.connection_type
```

Comment:
Should the type name be `Types::ProjectConnection` here?

Reply:
Not quite, `#connection_type` returns an instance of `GraphQL::ObjectType` whose name ends in `"Connection"`, so that's what triggers connection behavior here!